### ISSUE TITLE
Chia limiting ETH mining mh/s

### ISSUE THREAD

**Issue by dumpstar85** [Chia-Network/chia-blockchain#9681]
Hello everyone,

I've got a rig which has been happily mining ETH using a few 3080's however I noticed that whenever Chia is running on the same machine recently, it nerfs the MH/s of my first 3080 in pcie0 but none of the others.. 

Is there any reason why this program should be using this much power without any valid reason??

**Comment by brause** [comment]
well, i guess you did use the GUI ?

**Comment by linming66** [comment]
You have so many things. Don't dig it, can't it be fine? Wouldn’t it be enough to buy another computer?

**Comment by Jacek-ghub** [comment]
Maybe that GPU shares PCI lines with drive used to hold chia dbs.

**Comment by dumpstar85** [comment]
I really dont understand it.. its the GUI yes, but it was running for about 5 months absolutely fine getting great mh/s on my primary card and then all of a sudden it dropped. I assume its down to NVIDIA drivers but I just dont know how to work around it. I guess @linming66 is right, i need a crappy old machine just to run my umpteen drives. 

@brause, do you know if I will have to replot my drives if I migrate to the terminal/dos equivalent? 

Thankyou all for your responses and I wish you and your families a safe and successful new year!

**Comment by brause** [comment]
@dumpstar85 no, absolutely not. No replotting necessary. You just have to start it from the command line.
cd chia-dir (where that is that depends on you os)
. ./activate
chia start all
this will start everything without the gui.
see:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#start 
for more options...
and this is also helpful if you have trouble finding the right chia executable.
https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)

Good luck!

**Comment by brause** [comment]
@linming66  there are many reasons each and everybody does it different. One wants to be maximal energy efficient. One wants to admin fewer pcs. One does not want to spend more money on the energy bill. And for some it is very tempting to do it in the cleverest of ways(regardless of the amount work involved). and there are many more. all are good in their way.
